### PR TITLE
use fixed tiny_tuple version

### PR DESCRIPTION
**Pull request by pauloacmartinez**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -4,34 +4,44 @@ cmake_policy(SET CMP0077 NEW)
...
 project(hsm VERSION 0.2.0 LANGUAGES CXX)
 
-include(cmake/CPM.cmake)
-CPMAddPackage(NAME tiny_tuple
-  GITHUB_REPOSITORY "APokorny/tiny_tuple"
-  GIT_TAG development)
-CPMAddPackage("gh:TheLartians/PackageProject.cmake@1.6.0")
+include(cmake/get_cpm.cmake)
+CPMAddPackage(
+    NAME tiny_tuple
```
Comment (APokorny) [inline CMakeLists.txt:9]: why the change in indent?

File: example/alg_test.cpp [hunk 2]
```diff
@@ -1,8 +1,8 @@
 #define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
 
 #include <iostream>
-#include "catch.hpp"
-#include <hsm/hsm.hpp>
+#include "catch2/catch.hpp"
+#include "hsm/hsm.hpp"
```
Comment (APokorny) [inline example/alg_test.cpp]: keep the < here
From the perspective of the `example` code hsm and now with the external catch2 are libraries in use..

**Review by APokorny — CHANGES_REQUESTED** [review]

File: example/alg_test.cpp (hunk 2 above)
Comment (pauloacmartinez) [reply example/alg_test.cpp]: I restored the use of <

File: CMakeLists.txt (hunk 1 above)
Comment (pauloacmartinez) [reply CMakeLists.txt:9]: I increased the indent from 2 to 4 spaces for readability, I also did the same on tiny_tuple

**Comment by APokorny** [comment]
I will merge and clean up the mess aftwards..
